### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by tireddy2**

### CODE REVIEW

File: draft-ietf-add-resolver-info.md [hunk 1]
```diff
@@ -87,19 +87,15 @@ Encrypted DNS resolver:
 
 Reputation:
 : "The estimation in which an identifiable actor is held, especially by the
-   community or the Internet public generally" ({{Section 1 of ?RFC7070}}.
+   community or the Internet public generally" {{Section 1 of ?RFC7070}}.
```
Comment (boucadair) [inline draft-ietf-add-resolver-info.md]:
```suggestion
   community or the Internet public generally" ({{Section 1 of ?RFC7070}}).
```

File: draft-ietf-add-resolver-info.md [hunk 2]
```diff
@@ -87,19 +87,15 @@ Encrypted DNS resolver:
...
    A DNS client that wants to retrieve the resolver information may
-   use the RR type "RESINFO" defined in this document.
-
-   The content of the RDATA in a response to a query for RESINFO RR QTYPE is defined in
-   {{key-val}}.  If the resolver understands the RESINFO RR type, the
-   RRSet in the Authority section MUST have exactly one record. RESINFO is a property of the resolver
-   and is not subject to recursive resolution.
-
+   use the RR type "RESINFO" defined in this document. The content of the RDATA in a 
+   response to a query for RESINFO RR QTYPE is defined in {{key-val}}.
```
Comment (boucadair) [inline draft-ietf-add-resolver-info.md]:
```suggestion
   response to a query for RESINFO RR QTYPE is defined in {{key-val}}. RESINFO is a property of the resolver and is not subject to recursive resolution.
```

File: draft-ietf-add-resolver-info.md [hunk 3]
```diff
@@ -109,9 +105,9 @@ Reputation:
    using the RESINFO RR type and QNAME of "resolver.arpa". In this case, a client has to contend
    with the risk that a resolver does not support RESINFO. The resolver might
    pass the query upstream, and then the client can receive a positive RESINFO response either
-   from a legitimate DNS resolver or an attacker. The DNS client MUST set the Recursion
-   Desired (RD) bit of the query to 0 to ensure that the response is provided by the resolver.
```
Comment (boucadair) [inline draft-ietf-add-resolver-info.md:113]: Shoudn't we keep this part on RD?

**Review by boucadair — CHANGES_REQUESTED** [review]

File: draft-ietf-add-resolver-info.md (hunk 3 above)
Comment (tireddy2) [reply draft-ietf-add-resolver-info.md:113]: It is discussed in the WG that setting the RD flag will not solve the problem, see https://mailarchive.ietf.org/arch/msg/add/jZUYZrgqcmTXa6xZlYqNOZS-7wA/

**Review by boucadair — APPROVED** [review]

**Pull request merged**